beest/cmd/prepare.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"beest/cmd/driver"
+	"beest/cmd/step"
+	"github.com/spf13/cobra"
+)
+
+var prepareCmd = &cobra.Command{
+	Use:   "prepare [scenario]",
+	Short: "Deploy the integrations configured as part of the yard",
```

Comment:
Let's use the same bee terminology here to make it consistent. Can't remember if this is done for Bees or Hives? 😄

Reply:
agreed